src/01-functions.js

```diff
@@ -24,7 +33,10 @@ function alwaysTrue() {}
  *  greet("Ian");
  *  //> "Hello, Ian."
  */
-function greet() {}
+function greet(name) {
+  name = "Reed";
```

Comment:
Looks like we're hard coding the `name` Reed here instead of using the value passed in the parameters for name. I think if we omitted the re-assignment of name and just added a period to our interpolated string, these test would pass.